src/overlays/actors/ovl_En_Okarina_Tag/z_en_okarina_tag.c

```diff
@@ -30,14 +29,123 @@ const ActorInit En_Okarina_Tag_InitVars = {
     (ActorFunc)NULL,
 };
 
-#endif
+void EnOkarinaTag_Destroy(Actor* thisx, PlayState* play) {
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Okarina_Tag/EnOkarinaTag_Destroy.s")
+void EnOkarinaTag_Init(Actor* thisx, PlayState* play) {
+    EnOkarinaTag* this = THIS;
+    f32 zRot;
+    s32 i;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Okarina_Tag/EnOkarinaTag_Init.s")
+    zRot = 0.0f;
+    i = 0;
```

Comment:
(ugh, github wont let me open a suggestion because it splits a removal line...)

You can probably move the `zRot = 0.0f;` line up on top of the declared line `f32 zRot;` to make one line:
`f32 zRot = 0.0f` 
so long as you do the same thing for the `s32 i` below it.